manifests/init.pp

```diff
@@ -172,11 +172,19 @@
               }
             }
           }
+          case $::architecture {
+            /^i386/: {
+              $_architecture = 'i586'
```

Comment:
Please use architecture_real instead of _architecture to follow the rest of the module.